processor/probabilisticsamplerprocessor/logsprocessor_test.go

```diff
@@ -98,7 +98,7 @@ func TestLogsSampling(t *testing.T) {
 				SamplingPercentage: 50,
 				AttributeSource:    traceIDAttributeSource,
 			},
-			received: 52,
+			received: 45, // 52
```

Comment:
What does the comment mean?

Reply:
The comment is not needed. Good catch, thank you. I removed it.